nsv13/code/modules/overmap/fighters/fighters.dm

```diff
@@ -458,6 +457,26 @@ You need to fire emag the fighter's IFF board. This makes it list as "ENEMY" on
 			escape_pod = EP
 			EP.flight_state = NO_IGNITION
 
+/obj/structure/overmap/fighter/MouseDrop_T(mob/living/M, mob/user) //Passengers get in via dragging themselves

```

Comment:
Does this account for canopy being open? canopy should probably have to be open to hop in.

Reply:
Doesn't account, assuming separate hatch.

Reply:
Made it check for canopy being open.